package/gluon-mesh-vpn-tunneldigger/files/lib/gluon/upgrade/400-mesh-vpn-tunneldigger

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/lua
```

Comment:
All Lua scripts should be moved from `files` to `luasrc`, and `GluonSrcDiet` (and luadest CP) calls be added to the corresponding Makefiles, as seen in most Gluon packages.